library/std/src/lib.rs

```diff
@@ -258,6 +258,9 @@
 #![feature(staged_api)]
 #![feature(thread_local)]
 #![feature(try_blocks)]
+#![feature(abi_efiapi)]
+#![feature(exclusive_range_pattern)]
+#![feature(const_option)]
```

Comment:
const_option is a language feature originating from libcore and as such will need to be in the block below.